xml/systemd.xml

```diff
@@ -1485,51 +1483,78 @@ UMask=007</screen>
     <sect2 xml:id="sec-boot-systemd-custom-drop-in">
       <title>Creating drop-in files</title>
       <para>
-        For minor changes of a configuration file, use so-called drop-in files.
-        Drop-in files let you extend the configuration of unit files without
-        having to edit or override the unit files themselves.
+        For minor changes of a configuration file, use so-called drop-in files
```

Comment:
Maybe drop _minor_?
As a drop-ins may be the preferred way of config modification in the future.
Does it make sense, @fbuihuu ?